boards/feather-nrf52840/doc.txt

```diff
@@ -35,17 +35,26 @@ use an external Segger J-Link Programmer connected to the [SWD Connector].
 #### Flashing the uf2 bootloader
 
 To flash the uf2 bootloader (if its no longer present on your BOARD) then with
-a jlink attached to your BOARD and `nrf52prog` installed:
-
-$ git clone git@github.com:adafruit/Adafruit_nRF52_Bootloader.git
-$ cd Adafruit_nRF52_Bootloader
-$ make BOARD=feather_nrf52840_express all
-$ make BOARD=feather_nrf52840_express flash
+a jlink attached to your BOARD and [`nrfjprog`][nrfjprog] installed:
+
+~~~~~~~~~~~~~{.sh}
+git clone git@github.com:adafruit/Adafruit_nRF52_Bootloader.git
+cd Adafruit_nRF52_Bootloader
+# only download what is needed, use
+#     `git submodule update --init --recursive`
+# to download everything
+git submodule update --init -- lib/nrfx lib/tinyusb lib/uf2
```

Comment:
Ah just always download all submodules, otherwise we run the risk of having this doc become outdated again if the upstream repo structure changes / adds new dependencies

Reply:
But it takes ages, since `lib/tinyusb` has quite a lot submodules (who, I believe also have submodules).

Reply:
Looks like those are not needed though… 

    git submodule update --init

was enough for me to build the bootloader (no `--recursive`)

Reply:
Not for me... See line below.

Reply:
But yeah, maybe we can remove the paths :-).